### ISSUE TITLE
Failed to read the 'sessionStorage' property from 'Window': Access is denied for this document.

### ISSUE THREAD

**Issue by marjanaet** [wellyshen/use-places-autocomplete#807]
# Bug Report

Failed to read the 'sessionStorage' property from 'Window': Access is denied for this document.
Sometimes the suggestions are not showing and console logs this error. `Failed to read the 'sessionStorage' property from 'Window': Access is denied for this document.`

No clear steps to reproduce.

## Expected Behavior
It should work even session storage throws an error

## Screenshots
![Screen Shot 2021-07-26 at 9 07 03 AM](https://user-images.githubusercontent.com/62867750/126930097-781ba5a3-bc04-441e-ade1-628b0382693b.png)

## Your Environment
- Browser: [Chrome Mobile]
- Version 87.0.4280
- Operating System - Android
- Version 8.1.0

**Comment by wellyshen** [comment]
@marjanaet Did you work with iFrame?

**Comment by marjanaet** [comment]
@wellyshen No. This is from an android device. Not within an iframe.

**Comment by marjanaet** [comment]
In my opinion it should not depend on the session storage. Some devices/browsers may blocks the usage of session storage. In those times also this should work.

**Comment by wellyshen** [comment]
@marjanaet Do you know the reason for the blocking issues?

**Comment by marjanaet** [comment]
It should be due to browser settings.
https://stackoverflow.com/questions/58152236/what-to-do-when-sessionstorage-access-denied-due-to-browser-settings might help

**Comment by wellyshen** [comment]
@marjanaet So what you expect is that the cached data should work even if the session storage is blocked right?

**Comment by marjanaet** [comment]
Yes. the app crashes when the session storage is unavailable. If session storage is unavailable and  If we can get the app work without the cache it will be great.

**Comment by wellyshen** [comment]
@marjanaet Okay, I got it. Will fix it soon, thanks.

**Comment by wellyshen** [comment]
@marjanaet I don't have the reproduced device, can you verify this PR (#812) for me?

**Comment by marjanaet** [comment]
@wellyshen Thanks for looking in to this so quickly. Appreciate it.
Actually I too do not have a real device to check. But I was able to simulate this by blocking all cookies from the chrome settings.

First I tested on this master branch by blocking all cookies. Issue was reproduciable.
Then I switched to `fix/session-storage-crashed` branch with the setting for blocking all cookies. It is working fine.

Approving the fix.

Again appreciate for looking into this so quick! 👍 👍
